route/here/measure.go

```diff
@@ -94,13 +94,23 @@ func NewClient(apiKey string, opts ...ClientOption) Client {
 	return c
 }
 
+func cleanPoints(points []route.Point) []route.Point {
```

Comment:
This modifies in place. I think it would be better if we made a copy and clean that.

Reply:
yes, made a copy first